pkg/node/node.go

```diff
@@ -149,7 +149,8 @@ type Options struct {
 	ResolverConnectionCfgs     []multiresolver.ConnectionConfig
 	RetrievalCaching           bool
 	BootnodeMode               bool
-	SwapEndpoint               string
+	SwapEndpoint               string // deprecated: use rpc endpoint instead
+	RpcEndpoint                string
```

Comment:
Could `SwapEndpoint` be removed from `Options` as well?

If we are deprecating this option we should stop using it in all packages other then `cmd` were we need to keep it for backward compatibility.

Reply:
Yes, we can.